src/icons/SealSad.tsx

```diff
@@ -30,7 +33,7 @@ export default function ({
         height="65"
         viewBox="0 0 80 65"
         xmlns="http://www.w3.org/2000/svg"
-        className={displayFrom('md')}
+        className={noMobile ? undefined : displayFrom('md')}
```

Comment:
What if you use `!noMobile && displayFrom('md')`?
Also, on line 77 you use `display('hidden')` if the prop is `noMobile`. Should you use the same structure here?

Reply:
On `L77` I have little Seal which should be displayed only on mobiles (used for burner popup, for example)

Reply:
And this won't work unfortunately
![image](https://user-images.githubusercontent.com/49658988/196526209-4cfe0f3b-90ec-4140-a003-a98c4ae5e70a.png)

Reply:
Maybe our `displayFrom` and `displayTo` could consume the `{}` as prop 🤔